src/integrationtests/resources/com/aws/greengrass/integrationtests/logmanager/smallSpaceManagementPeriodicIntervalConfig.yaml

```diff
@@ -11,6 +11,7 @@ services:
             diskSpaceLimit: '105'
             diskSpaceLimitUnit: 'KB'
             deleteLogFileAfterCloudUpload: 'true'
+      periodicUploadIntervalSec: 1
```

Comment:
Given now releasing disk space runs after the upload I wanted to make this low for the test